### ISSUE TITLE
Read text messages

### ISSUE THREAD

**Issue by chfritz** [chfritz/typhone#5]

**Comment by gravypod** [comment]
Is this feature implemented or going to be implemented any time soon? I'd love being able to setup a service like this.

**Comment by chfritz** [comment]
No, this isn't implemented yet. I don't think I'll be able to add it any time soon. But pull requests are very welcome.
